training_web_react/src/Containers/MainPage/App.js

```diff
@@ -11,20 +13,64 @@ const App =(props) =>{
 
   const soundFail = "src/sounds/kick.wav";
   const soundHit = "src/sounds/hihat.wav";
+  const MAX_RANKING_SIZE=5;
+  const minPeepTime= 200;
+  const maxPeepTime= 1000;
+  const GAME_DURATION=10000;
+  const playSoundFail = new Audio(soundFail)
+  const playSoundHit = new Audio(soundHit)
+  const NEW_LEVEL = 5; // AMOUNT OF POINTS NEEDED TO INCREASE YOUR LEVEL
+  const SPEED_REDUCTION = 0.05; // MORE LVL
+
 
+  /************************************
+  *************************************
+  **************HOOKS******************
+  *************************************
+  *************************************
+  */
 
   const [score,setScore] = useState(0)
   const [holes] = useState(9)
   const [initGame,setInitGame] = useState(false)
   const [lastHole,setLastHole] = useState(0)
-  const playSoundFail = new Audio(soundFail)
-  const playSoundHit = new Audio(soundHit)
+  const [PlayerName, setPlayerName] = useState('')
+  const [DisableFooter,SetDisableFooter] = useState(false)
+  const [actualLevel,setActualLevel] = useState(0)
 
-  const minPeepTime= 200;
-  const maxPeepTime= 1000;
   
+
   const startPeep = useRef(false);
 
+  useEffect(()=>{
+    startPeep.current = initGame;
+    if(initGame){
+      peep()
+    }
+  },[initGame])
+
+  useEffect(()=>{
+    if(!initGame && score>0){
+      addToStore();
+      if(rankingStore.getState().length>MAX_RANKING_SIZE){
+          removeFromStore();
+      }
+    }
+  },[initGame,score,PlayerName])
+
+  useEffect(()=>{
+    if(score>0 && score%NEW_LEVEL===0){
+      setActualLevel(previus => previus+1)
+    }
+  },[score])
+
+
+  /************************************
+  *************************************
+  ************FUNCTIONS****************
+  *************************************
+  *************************************
+  */
```

Comment:
This type of comments are not necessary please remove, keep them (functions, hooks...) ordered like you do now  in groups but remove the comments